src/components/SavedProvidersList/saved-providers-list.js

```diff
@@ -50,6 +50,27 @@ function printSavedProviders(providers) {
   printJSX(printPage);
 }
 
+function emailSavedProviders(providers) {
+  const byTypeName = _.groupBy(providers, provider => provider.typeName);
+  let emailBodyString = ''
+  const newLine = "%0D%0A"
+  _.forEach(byTypeName, (providers, typeName) => {
+    let providerString = ''
+    providerString = providerString.concat(typeName + newLine)
```

Comment:
Looks great! this feature will be very useful. just some small nits:
- You can use += instead of concat, which is apparently faster: https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/String/concat#Performance
- You can use a multi-line template string to format the provider info (https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Template_literals).